WordPress/Classes/Services/MediaCoordinator.swift

```diff
@@ -587,7 +593,13 @@ extension MediaCoordinator: MediaProgressCoordinatorDelegate {
     func mediaProgressCoordinatorDidFinishUpload(_ mediaProgressCoordinator: MediaProgressCoordinator) {
         // We only want to show an upload notice for uploads initiated within
         // the media library, or if there's been a failure.
-        if mediaProgressCoordinator == mediaLibraryProgressCoordinator || mediaProgressCoordinator.hasFailedMedia {
+        // If the errors are causes by a missing file, we want to ignore that too.
+
+        let mediaErrorsAreMissingFilesErrors = mediaProgressCoordinator.failedMedia.allSatisfy { $0.hasMissingFileError }
+
+        if !mediaErrorsAreMissingFilesErrors,
+            mediaProgressCoordinator == mediaLibraryProgressCoordinator || mediaProgressCoordinator.hasFailedMedia {
```

Comment:
Why are we not checking hasFailedMedia at the beginning of this function and only if it does handle accordingly?

Reply:
This method is also responsible for presenting success notices.

Reply:
Yep. so I was thinking `if mediaProgressCoordinator.hasFailedMedia { (handle failure) }  else { handle success }`